src/controllers/UserController.js

```diff
@@ -26,4 +26,41 @@ export default class UserController {
       return res.status(500).json({ error: error.message });
     }
   }
+
+  /**
+ * @name getUser
+ * @description Allows user to sign in user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async getUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await UserServices.loginUser(email);
```

Comment:
I don't understand why you decided to create the `loginUser` method in the userServices file cause you are calling methods that already exist in the same file....which i don think its necessary. you can just use them in your controller since the methods already exist

Reply:
Rename this method to describe what it should be doing, which is to login a user

Reply:
I have done that